src/main/resources/application.properties.example

```diff
@@ -1,7 +1,7 @@
 spring.datasource.url=${TGB_APP_DB_URL}
 spring.datasource.username=${TGB_APP_DB_USERNAME}
 spring.datasource.password=${TGB_APP_DB_PASSWORD}
-spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
ezt miert csereltuk ki? az exampleban boven jo a `create-drop`

Reply:
Hi, indentation corrected. We made some print screens for Judit's presentation, that is why the extra line brakes. Deleted & corrected. IntelliJ deletes the Application.properties file when master pulled - magic. Example file set to update DB due easy copy-paste routine.

Reply:
ok

Reply:
in the readme then this should be mentioned